zone/entity.cpp

```diff
@@ -4582,6 +4582,26 @@ void EntityList::SendUntargetable(Client *c)
 	}
 }
 
+void EntityList::SendAppearanceEffects(Client *c)
+{
+	if (!c)
+		return;
+
```

Comment:
for (const auto& mob : mob_list) that way you don’t have to increment manually.

Reply:
I am fine with how I wrote this.